src/ASWF-Digital-Assets-1.0.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="ASWF-Digital-Assets-1.0"
+   name="ASWF Digital Assets License version 1.0" listVersionAdded="1.0">
```

Comment:
`listVersionAdded` is intended to be the version of the SPDX License List release in which this license is first added. The upcoming release is v3.21, so this should be "3.21".

Reply:
Thanks @swinslow, I will make that change right away